lib/chef/provider/package/rubygems.rb

```diff
@@ -401,7 +401,7 @@ def initialize(new_resource, run_context=nil)
         end
 
         def is_omnibus?
-          if RbConfig::CONFIG['bindir'] =~ %r!/opt/(opscode|chef)/embedded/bin!
+          if RbConfig::CONFIG['bindir'] =~ %r!/opt/(opscode|chef|chefdk)/embedded/bin!
```

Comment:
Hrm, this will be tricky if people relocate the install on windows. Should we just use `ANYTHING/embedded/bin` as the heuristic? @tyler-ball does `RbConfig::CONFIG['bindir']` normalize the path separator to forward slash?